scripts/generateDocs.js

```diff
@@ -1,20 +1,20 @@
 const path = require('path');
-const chalk = require('chalk');
 const markdownMagic = require('markdown-magic');
 const fs = require('fs');
+const https = require('https');
 
-const config = {
-  matchWord: 'AURO-GENERATED-CONTENT'
-}
+const readmeTemplateUrl = 'https://raw.githubusercontent.com/AlaskaAirlines/WC-Generator/master/componentDocs/README.md';
+const dirDocTemplates = './docTemplates';
+const readmeFilePath = dirDocTemplates + '/README.md';
```

Comment:
Please revert this whole file. Many of the features in this script this repo does not support. 

https://github.com/AlaskaAirlines/auro-hyperlink/blob/main/scripts/generateDocs.js